gensrc/thrift/InternalService.thrift

```diff
@@ -294,6 +294,7 @@ struct TExecPlanFragmentParams {
   14: optional TLoadErrorHubInfo load_error_hub_info
 
   50: optional bool is_pipeline
+  51: optional i32 pipeline_dop
```

Comment:
needn't add here. Use query_options. pipeline_dop is enough.

Reply:
ok

Reply:
pipeline_dop is now is FragmentInstance-specific parameter, QueryOptions contain uniform parameters for all FragmentInstances underneath a query.